.github/workflows/test.yaml

```diff
@@ -3,53 +3,12 @@ on:
   push:
     branches:
     - master
+    - unity
   pull_request:
     branches:
     - master
+    - unity
 jobs:
-  test:
```

Comment:
Why are we removing these tests?

Reply:
These ran with go-acc which tries to upload coverage to some service and complained that we don't have an upload token. I think test i-386 is sufficient (it simply does go test ./...) and didn't bother trying to fix the other ones.

Reply:
Ahh ok, no problem then.